### PR TITLE
Add Tagesspiegel

### PR DESCRIPTION
**Pull request by fabianhenning**

### CODE REVIEW

File: src/fundus/publishers/de/tagesspiegel.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            return self.precomputed.meta.get("og:title")
+
+        @attribute
+        def publishing_date(self) -> Optional[datetime.datetime]:
+            return generic_date_parsing(self.precomputed.ld.bf_search("datePublished"))
+
+        @attribute
+        def free_access(self) -> bool:
+            paywall_headline = self.precomputed.doc.cssselect(".paywall-headline")
+            return len(paywall_headline) == 0
```
Comment (MaxDall) [inline src/fundus/publishers/de/tagesspiegel.py:41]: This one is obsolete and automatically extracted during parsing.

Additional: Tagespiegel supports `topics` accessible with the `keywords` key within the articles `LD`

File: src/fundus/publishers/de/tagesspiegel.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+
+class TagesspiegelParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("div[id=story-elements] > p")
+        _subheadline_selector = CSSSelector(".Ha6")
```
Comment (MaxDall) [inline src/fundus/publishers/de/tagesspiegel.py:18]: It looks like the newspaper also supports a `summary`.

Further the subheadline selector does not seem to work for most cases.

**Review by MaxDall — CHANGES_REQUESTED** [review]
Thanks for adding :+1:

File: src/fundus/publishers/de/tagesspiegel.py (hunk 1 above)
Comment (fabianhenning) [reply src/fundus/publishers/de/tagesspiegel.py]: Thank you! I removed the redundant free_access and added topics

File: src/fundus/publishers/de/tagesspiegel.py (hunk 2 above)
Comment (fabianhenning) [reply src/fundus/publishers/de/tagesspiegel.py]: added summary and fixed the subheadline selector

**Review by MaxDall — DISMISSED** [review]
Looks really nice 👍 Thanks a lot

**Review by addie9800 — APPROVED** [review]

**Pull request merged**